cdap-app-templates/cdap-etl/cdap-etl-batch/src/test/java/co/cask/cdap/etl/batch/ETLSparkTestRun.java

```diff
@@ -1,102 +0,0 @@
-/*
- * Copyright © 2015 Cask Data, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not
- * use this file except in compliance with the License. You may obtain a copy of
- * the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations under
- * the License.
- */
-
-package co.cask.cdap.etl.batch;
-
-import co.cask.cdap.api.data.format.StructuredRecord;
-import co.cask.cdap.api.data.schema.Schema;
-import co.cask.cdap.api.dataset.table.Table;
-import co.cask.cdap.etl.mock.batch.MockSink;
-import co.cask.cdap.etl.mock.batch.MockSource;
-import co.cask.cdap.etl.mock.transform.StringValueFilterTransform;
-import co.cask.cdap.etl.proto.Engine;
-import co.cask.cdap.etl.proto.v2.ETLBatchConfig;
-import co.cask.cdap.etl.proto.v2.ETLStage;
-import co.cask.cdap.proto.artifact.AppRequest;
-import co.cask.cdap.proto.id.ApplicationId;
-import co.cask.cdap.proto.id.NamespaceId;
-import co.cask.cdap.test.ApplicationManager;
-import co.cask.cdap.test.DataSetManager;
-import co.cask.cdap.test.WorkflowManager;
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ImmutableSet;
-import com.google.common.collect.Sets;
-import org.junit.Assert;
-import org.junit.Test;
-
-import java.util.Set;
-import java.util.concurrent.TimeUnit;
-
-/**
- *
- */
-public class ETLSparkTestRun extends ETLBatchTestBase {
-
-  @Test
-  public void test() throws Exception {
-    /*
-     *            ---------------- sink1
-     *           |
-     * source ---
-     *           |
-     *            ---- filter ---- sink2
-     */
-    ETLBatchConfig etlConfig = ETLBatchConfig.builder("* * * * *")
-      .setEngine(Engine.SPARK)
-      .addStage(new ETLStage("source", MockSource.getPlugin("sparkinput")))
-      .addStage(new ETLStage("sink1", MockSink.getPlugin("sparkoutput1")))
-      .addStage(new ETLStage("sink2", MockSink.getPlugin("sparkoutput2")))
-      .addStage(new ETLStage("filter", StringValueFilterTransform.getPlugin("name", "samuel")))
-      .addConnection("source", "sink1")
-      .addConnection("source", "filter")
-      .addConnection("filter", "sink2")
-      .build();
-
-    AppRequest<ETLBatchConfig> appRequest = new AppRequest<>(APP_ARTIFACT, etlConfig);
-    ApplicationId appId = NamespaceId.DEFAULT.app("DagApp");
```

Comment:
this seemed to be covered by existing tests in DataPipeline so I didn't move it.